### PR TITLE
Added files from my Microverse trials to be reviewed

### PR DESCRIPTION
**Pull request by yonas44**
Added index.html 
Added additional files need 

Thankyou for reviewing my PR!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <!-- <h1>Hello, World!</h1> -->
+    <!-- <p>Hello, am excited to be in the Microverse</p> -->
```
Comment (yonas44) [inline index.html:26]: I think it would be better if you removed unwanted lines of code, that way you can have cleaner code.👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (yonas44) [inline index.html:3]: I think its better to remove the extra free space in your code, as it creates confusion rather its better implementing it in the right time and place.👍

**Review by yonas44 — COMMENTED** [review]
Good work on using Semantic tags✔

File: index.html (hunk 1 above)
Comment (Olamarx) [reply index.html:26]: [OPTIONAL] Kindly make sure you drag all the aspect you are commenting on, this is for the student to know the area you are specifying.

**Review by Olamarx — APPROVED** [review]
Hi @yonas44,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good use of Github review feature ✔️
- Grammatically correct review ✅ 
- Polite choice of words ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (yonas44) [reply index.html:26]: Okay I will do that, thankyou for your feedback! @Olamarx

**Pull request merged**